universal-login-wallet/src/ui/react/ConnectAccount/ConnectWithPassphrase.tsx

```diff
@@ -11,13 +11,14 @@ export const ConnectWithPassphrase = ({name}: ConnectWithPasssphraseProps) => {
   const [code, setCode] = useState('');
   const [isLoading, setIsLoading] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
-  const {walletService} = useServices();
+  const {walletService, walletStorageService} = useServices();
   const {history} = useRouter();
 
   const onRecoveryClick = async () => {
     setIsLoading(true);
     try {
       await walletService.recover(name, code);
+      walletStorageService.save();
```

Comment:
Should be just one line: 
await walletService.recover(name, code);
#reversedependencies